packages/apps/tools/src/components/Global/AddNetworkModal/AddNetworkModal.tsx

```diff
@@ -120,7 +120,11 @@ export const AddNetworkModal: FC<IAddNetworkModalProps> = (props) => {
                   </Stack>
                 </section>
                 <section className={formButtonStyle}>
-                  <Button type="submit" icon="TrailingIcon">
+                  <Button
+                    type="submit"
+                    icon="TrailingIcon"
+                    disabled={Boolean(error)}
```

Comment:
Out of curiosity; why was this done / why is this needed? 😁

Reply:
To prevent submitting if there are errors, such as duplicate networks etc

Reply:
Thanks for the explanation 👍